docs/contributing.md

```diff
@@ -3,12 +3,9 @@ Any contribution should observe the convention set fourth in the [Code Tour](cod
 The following must complete without error prior to making a pull request:
 
 1. `gulp vet`
-2. `gulp test`
+2. `yarn test`
 
 ## Testing
 When adding or modifying functionality, unit and integration tests must accompany any changes.
 Examples of both state and component tests can be found in `tests/`.
 If any notional data is required for the test ensure an appropriately titled folder is created in `tests/mock/`.
-
-The `gulp autotest` task can be used to automatically run the test suite as you
```

Comment:
What do you think about converting `gulp autotest` to something like `yarn autotest` or `yarn run autotest`? 

I'm not sure how much extra effort it would be but this task could take advantage of webpack incremental builds so it wouldn't take so long for each test run.